test/integration/master/audit_test.go

```diff
@@ -269,3 +344,134 @@ func expectNoError(t *testing.T, err error, msg string) {
 		t.Fatalf("%s: %v", msg, err)
 	}
 }
+
+// the following functions are copied from test/integration/apiserver/admissionwebhook
```

Comment:
copying this code is a blocker for the PR. Please move code first where is can be reused.